opencensus/proto/exporter/exporter.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opencensus.proto.exporter;
+
+import "opencensus/proto/trace/trace.proto";
+import "opencensus/proto/stats/metrics/metrics.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.exporter";
+option java_outer_classname = "ExporterProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/exporterproto";
+
+service Export {
+  rpc Export (ExportRequest) returns (ExportResponse) {}
```

Comment:
To not have to start an RPC every request I would suggest to use bi-directional streams:
rpc Export (stream ExportRequest) returns (stream ExportResponse) {}